lib/commands/settings.js

```diff
@@ -1,4 +1,6 @@
+import { errors } from 'appium-base-driver';
 import log from '../logger';
+import { youiEngineDriverReturnValues } from '../utils';
```

Comment:
Looking at the error codes in `util` I wonder if these are defined somewhere in Appium?